src/CMakeLists.txt

```diff
@@ -14,6 +14,12 @@ configure_file (${CMAKE_SOURCE_DIR}/include/fenix-config.h.in
 configure_file(${CMAKE_SOURCE_DIR}/include/fenix.h
   "${CMAKE_BINARY_DIR}/include/fenix.h" COPYONLY)
 
+configure_file(${CMAKE_SOURCE_DIR}/include/fenix.h
+  "${CMAKE_BINARY_DIR}/include/fenix_constants.h" COPYONLY)
+
+configure_file(${CMAKE_SOURCE_DIR}/include/fenix.h
```

Comment:
Wow, this was very sloppy of me, and I don't know why this worked--though it did. Let me quickly fix.